styles/style.css

```diff
@@ -1,31 +1,89 @@
+.container {
+  width: 100%;
+  margin: 0 auto;
+}
 h1 {
   text-align: center;
+  margin-top: 120px;
+  font-family: roboto;
+  color: #383838;
 }
 hr {
   border: 0;
   background-color: #f5bd41;
   width: 100px;
   height: 4px;
 }
-/* TEXT */
-.text {
+.features {
   display: flex;
-  justify-content: center;
-  margin-top: 20px;
-  margin-right: 40px;
+  margin: 50px 0;
 }
-.text_item {
+.features_item {
   width: 20%;
+  text-align: center;
+  padding: 0 65px;
+  margin: 25px 0;
+  font-family: roboto;
+  line-height: 28px;
+}
+.features_icon {
+  margin-bottom: 50px;
+}
+.features_title {
   font-size: 24px;
   color: #383838;
 }
-/* ABOUT */
-.about {
+#lol {
+  margin-bottom: 67px;
+}
+
+@font-face {
```

Comment:
Please remove to **the begining of style**
And you added Roboto with `font-weight: normal`, but your link connects Roboto-Light!
Please use this
```
  font-family: 'Roboto';
  src: url("...woff2") format("woff2"),
  url("...woff") format("woff");
  font-weight: 400;
  font-style: normal;
```
And add second font with `font-weight: 500`
I recommend to use **woff** and **woff2** formats, if you don't need to support outdated browsers


**You need to add a special font for every font-weight and font-style, that will be used** 

If you add every type of font in your folder, it doesn't mean that your code will see this